package.json

```diff
@@ -114,5 +115,8 @@
       "dependencyPin": "Pin ${dependency} to ${oldVersion}",
       "devDependencyPin": "Pin ${dependency} to ${oldVersion}"
     }
+  },
+  "dependencies": {
+    "rdfxml-streaming-parser": "^1.0.0"
```

Comment:
This dependency should not be added on the root, only in `packages/actor-rdf-parser-rdfxml`.